### PR TITLE
Made bannered campaign minisite nav full width

### LINKED ISSUE #2916: Bannered campaign page sub nav width
Checking off "Narrowed page content:" checkbox should not change the width of the subnav.

Screen cap from prod:
image

It should be like the about page.
image

Can be grouped with https://github.com/mozilla/foundation.mozilla.org/issues/2769

cc @alanmoo @Pomax

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #2916

- **narrow** bannered campaign page: https://foundation-mofostaging-pr-2919.herokuapp.com/en/bannered-campaign-page/

### Shouldn't be affected

- **narrow** multipage campaign with petition: https://foundation-mofostaging-pr-2919.herokuapp.com/en/campaigns/multi-page/ 
- **full-width** multipage campaign with petition: https://foundation-mofostaging-pr-2919.herokuapp.com/en/campaigns/act-former-heavy/

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 {% block minisite_title_mb %}
   {% if uses_menu %}
+  {% block bannered_campaign_page_menu %}
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html]: can we call this something that "makes sense" from the mini site's perspective, which knows nothing about which pages extend from it? E.g. `minisite_title_menu`?

File: network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html [hunk 2]
```diff
@@ -8,3 +8,19 @@
 {% endblock %}
 
 {% block campaign_page_header %}{% endblock %}
+{% block campaign_page_menu %}{% endblock %}
+
+{% block bannered_campaign_page_menu %}
+{% comment %}
+  Wrapper HTML elements copied from primary_page_menu.html as we want menu on BanneredCampaignPage to look exactly the same as PrimaryPage's.
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html]: If you put this _above_ the block, then it's "not templating content" and you don't need the comment tag wrapping.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html (hunk 1 above)
Comment (mmmavis) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/mini_site_name_space.html]: Sure. Btw, what does `minisite_title` mean? Is it the "title" section on minisite?

**Comment by mmmavis** [comment]
Some unnecessary margin has been added to single page campaign page as side effect (thanks for Percy for catching it). I'll file a followup ticket for this.

image

**Comment by mmmavis** [comment]
follow-up ticket filed: https://github.com/mozilla/foundation.mozilla.org/issues/2920

**Pull request merged**